Tools/c-analyzer/c_parser/preprocessor/__init__.py

```diff
@@ -47,15 +49,23 @@ def preprocess(source, *,
     if tool:
         logger.debug(f'CWD: {os.getcwd()!r}')
```

Comment:
I think it might be useful to log `cwd` instead of os.getcwd() when cwd is not None.